### PR TITLE
[WIP] very preliminary start_query method for STMO class

### PR DESCRIPTION
**Pull request by teonbrooks**
getting familiar with the stmocli and the redash client.

### CODE REVIEW

File: stmocli/stmo.py [hunk 1]
```diff
@@ -18,6 +18,28 @@ def __init__(self, redash_api_key, conf=None):
         self._redash = RedashClient(redash_api_key)
         self.redash_api_key = redash_api_key
 
+    def start_query(self, file_name, data_source_id=1, description=None):
+        """Add a query created locally to .stmocli.conf.
```
Comment (tdsmith) [inline stmocli/stmo.py]: A little bikeshedding:
* Should this method also push the new query to STMO?
* `start` sounds a little bit like executing a query to me; maybe `create`?

**Review by tdsmith — COMMENTED** [review]
Hey! Sorry if this review is premature; this will be a really useful feature!

File: stmocli/stmo.py [hunk 2]
```diff
@@ -47,10 +69,11 @@ def track_query(self, query_id, file_name):
...
         """
-        query = self.get_query(query_id)
-        query_file_name = file_name(query) if callable(file_name) else file_name
-        with open(query_file_name, "w") as outfile:
-            outfile.write(query["query"])
+        if query_id:
+            query = self.get_query(query_id)
+            query_file_name = file_name(query) if callable(file_name) else file_name
+            with open(query_file_name, "w") as outfile:
+                outfile.write(query["query"])
```
[the remark this answers is not in the document]
Comment (tdsmith) [reply stmocli/stmo.py]: Hm, meant to highlight the new conditional starting at `if query_id:`

**Comment by teonbrooks** [comment]
going through my backlog. this pr is extremely stale. @tdsmith, @harterrt either of you still using stmocli? if so, would this be useful. otherwise, I think I might close it for the time being and clear it from my mental docket

**Comment by teonbrooks** [comment]
also, thanks @tdsmith for all the reviews, they're extremely useful. sorry I never followed up

**Comment by harterrt** [comment]
Eh, I'd say close it. Now that we've migrated to BigQuery we can use the BQ CLI to achieve most of what we were trying to fix here.

**Comment by tdsmith** [comment]
Yeah, I was going to say the same thing.

**Pull request closed without merging**